components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java

```diff
@@ -3080,6 +3111,14 @@ private List<IdentityProviderProperty> getCombinedProperties(JustInTimeProvision
         associateLocalUser.setName(IdPManagementConstants.ASSOCIATE_LOCAL_USER_ENABLED);
         associateLocalUser.setValue("false");
 
+        IdentityProviderProperty federatedAssociationProperty = new IdentityProviderProperty();
+        federatedAssociationProperty.setName(IdPManagementConstants.FEDERATED_ASSOCIATION_ENABLED);
+        federatedAssociationProperty.setValue("false");
+
+        IdentityProviderProperty lookupAttribute = new IdentityProviderProperty();
+        lookupAttribute.setName(IdPManagementConstants.LOOKUP_ATTRIBUTES);
+        lookupAttribute.setValue(IdPManagementConstants.DEFAULT_LOOKUP_ATTRIBUTE);
```

Comment:
As mentioned above, this seems not necessary

Reply:
Addressed with [3c20442](https://github.com/wso2/carbon-identity-framework/pull/5017/commits/3c20442145ac5549fb011ce7fae823d2970cfbe9)